uast/uast-kotlin-fir/src/org/jetbrains/uast/kotlin/internal/firKotlinInternalUastUtils.kt

```diff
@@ -33,6 +37,19 @@ internal fun KtType.toPsiClass(context: KtElement): PsiClass? {
     }
 }
 
+internal fun KtCallElement.toFunctionLikeSymbol(): KtFunctionLikeSymbol? {
```

Comment:
The same as in https://github.com/JetBrains/intellij-kotlin/pull/96#discussion_r684035608 . `KtSymbol` should not be leaked outside `analyse` call

Reply:
I can't come up with a better util signature, other than leaking `KtSymbol` :p So, it looks not quite necessary; I dropped the resolution utilizing commit as a whole.